EmailController.java

```diff
@@ -0,0 +1,74 @@
+package com.example.demo;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.http.MediaType;
+import org.springframework.web.bind.annotation.*;
+
+import java.util.ArrayList;
+import java.util.Date;
+import java.util.stream.Collectors;
+
+import static org.springframework.web.bind.annotation.RequestMethod.DELETE;
+import static org.springframework.web.bind.annotation.RequestMethod.GET;
+import static org.springframework.web.bind.annotation.RequestMethod.POST;
+
+@RestController()
+@RequestMapping("/Email")
+public class EmailController {
+
+    WorkWithJsonFile workWithJsonFile;
+
+    @Autowired
+    public EmailController(WorkWithJsonFile workWithJsonFile) {
+        this.workWithJsonFile = workWithJsonFile;
+    }
+
+    @RequestMapping(value = "/activeEmails", method=GET)
+    private ArrayList<Email> showsActiveEmails(){
+        return (ArrayList<Email>) workWithJsonFile.loadFromJsonEmailsList().stream()
+                                                        .filter(email -> email.getDeliveryDate().after(new Date()) && !email.getSended())
+                                                        .collect(Collectors.toList());
+    }
+
+    @RequestMapping(value = "/newEmail", method=POST,consumes = MediaType.APPLICATION_JSON_VALUE)
+    private void addNew(@RequestBody Email newEmail){
+        workWithJsonFile.saveEmailToJson(newEmail);
+    }
+
+    @DeleteMapping(value = "/delete")
```

Comment:
The url is the name and should not contain verbs. Request method is telling what to do(DELETE in your case). Also, when creating url for delete operation you need to specify unique url which identifies resource you're willing to delete. In this case an id would work fine:
`@DeleteMapping("/{id}")`